editor/editor_command_palette.cpp

```diff
@@ -235,6 +265,9 @@ EditorCommandPalette::EditorCommandPalette() {
 	search_options->set_hide_folding(true);
 	search_options->add_theme_constant_override("draw_guides", 1);
 	search_options->set_columns(2);
+	search_options->set_column_title(0, TTR("Name"));
+	search_options->set_column_title(1, TTR("Shortcut"));
```

Comment:
I don't see these names show up anywhere. You probably need to call `set_column_titles_visible(true)`, otherwise it's useless to set them.

Reply:
Yep.